io.catenax.material_flow_scenario_request/2.0.0/MaterialFlowScenarioRequestAspect.ttl

```diff
@@ -0,0 +1,213 @@
+   samm:characteristic :ScenarioOwnerRoleCharacteristic ;
+   samm:exampleValue "Customer" .
+
+:scenarioDescription a samm:Property ;
+   samm:preferredName "scenarioDescription"@en ;
+   samm:description "Description of the scenario"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Changes in Delivery Date" .
+
+:parameterId a samm:Property ;
+   samm:preferredName "parameterId"@en ;
+   samm:description "Identifier of scenario parameter"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "847c71e5-614a-468b-a3a0-674bf2af3004" .
+
+:parameterDeliveryDateInitial a samm:Property ;
+   samm:preferredName "parameterDeliveryDateInitial"@en ;
+   samm:description "Delivery date initial (without scenario changes), optional, NULL when initially not set"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-10-09T10:00:00.000Z"^^xsd:dateTime .
+
+:parameterDeliveryDateUpdated a samm:Property ;
+   samm:preferredName "parameterDeliveryDateUpdated"@en ;
+   samm:description "Delivery date updated (with scenario changes)"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-10-10T09:00:00.000Z"^^xsd:dateTime .
+
+:parameterQuantityInitial a samm:Property ;
+   samm:preferredName "parameterQuantityInitial"@en ;
+   samm:description "Amount initial (without scenario changes), optional, NULL, when initially not set"@en ;
+   samm:characteristic :QuantityCharacteristic ;
+   samm:exampleValue "1"^^xsd:float .
+
+:parameterQuantityUpdated a samm:Property ;
+   samm:preferredName "parameterQuantityUpdated"@en ;
+   samm:description "Amount updated (with scenario changes)"@en ;
+   samm:characteristic :QuantityCharacteristic ;
+   samm:exampleValue "1"^^xsd:float .
+
+:parameterOrderId a samm:Property ;
+   samm:preferredName "parameterOrderId"@en ;
+   samm:description "ID of the order"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "OID-011123546" .
+
+:parameterComment a samm:Property ;
+   samm:preferredName "parameterComment"@en ;
+   samm:description "Optional description of parameters"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "updated Delivery Date" .
+
+:resultOwnId a samm:Property ;
+   samm:preferredName "resultOwnId"@en ;
+   samm:description "ID of scenario result own"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "916b5688-8bd8-4d7e-83b9-e0d40939274e" .
+
+:resultOwnSimRunInitial a samm:Property ;
+   samm:preferredName "resultOwnSimRunInitial"@en ;
+   samm:description "Foreign key to simulation result based on initial parameters"@en ;
+   samm:characteristic :SimResult .
+
+:resultOwnSimRunUpdated a samm:Property ;
+   samm:preferredName "resultOwnSimRunUpdated"@en ;
+   samm:description "Foreign key to simulation result based on updated parameter (scenario)"@en ;
+   samm:characteristic :SimResult .
+
+:ScenarioOwnerCharacteristic a samm:Characteristic ;
+   samm:preferredName "ScenarioOwnerCharacteristic"@en ;
```

Comment:
please also add `description`

Reply:
Added a description for the characteristic